src/ga.js

```diff
@@ -175,11 +182,20 @@ function getCpmDistribution(cpm) {
   return distribution;
 }
 
+exports.prefixedSend = function(){
+  if (_trackerName !== null) {
```

Comment:
There really is no need to do this check repeatedly. If it's `null` then we can simply set the variable `_trackerName` to the default. If you want to make this work with your text, just export a variable called `trackerName`